spec/spec.md

```diff
@@ -367,6 +367,21 @@ https://github.com/trustoverip/tswg-keri-specification/issues/69
 https://github.com/trustoverip/tswg-keri-specification/issues/70
 :::
 
+[[def: Dead-Attack]]
+
+~ See (Security Properties of Prerotation)[#dead-attacks]
+
+[[def: Live-Attack]]
+
+~ See (Security Properties of Prerotation)[#live-attacks]
+
+[[def: First-Seen]]
+
+~ refers to the first instance of a [[ref: message]] recieved by any [[ref: witness]] or [[ref: watcher]]. The first-seen event is always seen, and can never be unseen. It forms the basis for [[ref: duplicity detection]] in KERI based systems.
+
+[[def: Key-State]]
+
+~ intitially key-state is establised through an inception event, containing information about the current keys, next keys, current and next thresholds and configutation information. Subsequent events alter the key-state.
```

Comment:
It seems like this is already defined on line 155 as `Includes the set of currently authoritative keypairs for an AID and any other information necessary to secure or establish control authority over an AID.`  which seems more appropriate as a definition.

Reply:
I quite like the newly proposed definition as it is more specific. The old definition seems to be more complete but is quite vague. Perhaps, they can be combined. Something like
```
~ a set of currently authoritative keypairs for an AID and any other information necessary to secure or establish control authority over an AID. This information includes current keys, next keys, current thresholds, next thresholds, and configuration. A key-state is established through an inception event and may be altered by subsequent rotation events.
```

Reply:
That seems reasonable to me too.